packages/platform/src/types/Mutation.ts

```diff
@@ -25,11 +25,13 @@ interface GenerateBaseMutationsArgs {
   reducers?: any
   schemas?: any
   inputTypes?: any
+  // TODO(JJ): is returning void right here? It should return a UserRole, no?
   roleAssigner?: (ix: number) => void
```

Comment:
yes, that is true, if it is set it should return a user role. if not set, the function is undefined